OptionSelector/OSCategoryTBV.m

```diff
@@ -0,0 +1,177 @@
+//
+//  OSCategoryTBV.m
+//  OptionSelector
+//
+//  Created by Jamaal Sedayao on 8/13/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OSCategoryTBV.h"
+#import "OSOptionsTBV.h"
+
+@interface OSCategoryTBV ()
+
+@end
+
+@implementation OSCategoryTBV
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.title = @"Categories";
+    
+    self.allCategories = [[NSMutableArray alloc]init]; //important. initialize the NSMutableArray or nothing appears when view loads
+    
+    CQCategory * buddies = [[CQCategory alloc] init]; //initialize instance of CQCategory class
+    
+    buddies.name = @"Hangout Buddies";
+    buddies.options = @[
```

Comment:
This is a good opportunity to create a separate abstract class that includes all this data, say called "ListData". This would be your "Model" in MVC. This class could have three methods that each return a CQCategory, something like dogsData, carsData, and gamesData. Then your viewDidLoad would look much small and not include the actual data:

self.categories = [[NSMutableArray alloc]init];

self.allCategories = @[
[ListData buddiesData],
[ListData foodData],
[ListData activitiesData]
];